Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.8.6-slim-buster as base
+
+RUN apt-get update; apt-get -y install curl; \
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python; \
+    mkdir /app
+WORKDIR /app
+EXPOSE 5000
+
+FROM base as development
+COPY requirements-dev.txt /app
+RUN pip3 install -r requirements-dev.txt; rm requirements-dev.txt
+
+ENTRYPOINT ["python", "/app/app.py"]
```

Comment:
Why `python app.py` instead of `flask run`?

Reply:
Changed this now

Reply:
(will be in the next push)